src/classes/CRLP_Query_SEL_TEST.cls

```diff
@@ -54,6 +54,7 @@ private class CRLP_Query_SEL_TEST {
         List<Account> accts = Database.query(soql);
         System.AssertEquals(0, accts.size(), 'There should be no results, but the query should work');
 
+        List<Id> contactIds = new List<Id>();
```

Comment:
I think this should include a note as to why the var is needed.

Reply:
I also think might be helpful to assert that the query string includes the additional WHERE filter against contactIds.